### PR TITLE
Improve probe dictionary-c++ display

### PR DESCRIPTION
**Pull request by shreya99oak**
#449 
@georgf  I changed the branch and made the changes, yet the other commits appear

### CODE REVIEW

**Comment by georgf** [comment]
@shreya99oak can we close #449, are you planning to continue working on this here?

**Comment by shreya99oak** [comment]
Yes I'll work on it here

File: index.html [hunk 1]
```diff
@@ -5,11 +5,10 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Telemetry portal</title>
-
+
```
Comment (chutten) [inline index.html]: Looks as though there might be some unrelated whitespace changes to this file? (git tip: try `git checkout origin/master -- path/to/file` to change a file back to an unedited version

Comment (shreya99oak) [reply index.html]: Okay

**Comment by shreya99oak** [comment]
Removed the whitespaces @chutten, pls. review

**Review by chutten — APPROVED** [review]
Looks good!

**Comment by chutten** [comment]
Unfortunately I cannot merge these changes due to conflicts. Could you rebase your changes atop `mozilla:gh-pages` and then resubmit?

**Comment by shreya99oak** [comment]
@chutten  so would that be running the command "git rebase gh-pages" when I'm on the my "probe" branch?

**Comment by chutten** [comment]
Only if your `probe` branch is set to track the upstream mozilla's `gh-pages` branch. Otherwise it will rebase your changes on top of whatever branch your `probe` branch is set to track.

If your `git remote -v` does not include the mozilla repository, I recommend adding it with something like `git remote add moz https://github.com/mozilla/telemetry-dashboard.git`

Then you can `git branch probe --set-upstream-to=moz/gh-pages` to tell git that you want your `probe` branch to track the `gh-pages` branch of the `moz` upstream repo.

Then, when on the `probe` branch, you can use `git pull --rebase` to first fetch the latest version of `moz/gh-pages` and then rebase your changes on top of it. (it is equivalent to `git fetch && git rebase`)

Does this help?

**Comment by shreya99oak** [comment]
Thanks for the help! @chutten
